### ISSUE TITLE
bring docklike-plugin to debian repository

### ISSUE THREAD

**Issue by freddii** [nsz32/docklike-plugin#56]
Are there any plans to bring docklike-plugin to the debian repo in future?

It would be nice to be able to install it with simply running:

    sudo apt-get install xfce4-docklike-plugin  #(like it is possible with xfce4-whiskermenu-plugin)

**Comment by nsz32** [comment]
Yes, we should at good time join the official repository : we are not 100% ready yet, some bugs ...
Also, for the translation credits (you posted DE), please see my last message @ #55

**Comment by unit193** [comment]
Howdy,

If anyone is actually interested in doing this, I'm the one that packaged it in the Xubuntu PPA so can either help, or get you started out.  Feel free to reach out, I'm specifically quick to answer if you happen to use IRC.

~Unit 193

**Comment by SwampRabbit** [comment]
Agree that this should be proposed as an official Xfce core panel component on their GitLab, that's the only way for this to have good longevity, and also ensures all distros can benefit from it.

The Debianization of docklike-plugin isn't the issue, was debianized back when it was nmt-plugin, its getting some dedicated and skilled contributors in my opinion to take the load off Nico.

Nico said in another issue that his time was limited, so really no better way than have the Xfce team help, or whats going to happen is each of us will continue to splinter fork it and package it for our own distros until it slowly dies off.

**Comment by philipzae** [comment]
The Xfce devs are okay with moving to gitlab, as opened in this issue by the Thunar dev. https://github.com/davekeogh/xfce4-docklike-plugin/issues/16

**Comment by freddii** [comment]
seams that docklike plugin is now an official part of xfce:
https://gitlab.xfce.org/panel-plugins/xfce4-docklike-plugin

but i can't find it in debian repo yet.
https://packages.debian.org/search?searchon=names&keywords=+xfce4-

**Comment by freddii** [comment]
this script is working for me:

`#!/bin/bash
#to run this script go into the folder where it is then run: ./install_xfce-docklike-plugin.sh
#
set -eu
#
folder_name="xfce-docklike-plugin"
#
sudo apt install libxfce4panel-2.0-dev libwnck-3-dev xfce4-dev-tools libxfce4ui-2-dev -y
#
if [ -d "/home/$USER/programs_setup/$folder_name" ]; then
  cd /home/$USER/programs_setup/xfce-docklike-plugin/docklike-plugin
  sudo make uninstall
  cd /home/$USER/
  sudo rm -rf /home/$USER/programs_setup/xfce-docklike-plugin
fi
#
mkdir -p /home/$USER/programs_setup/xfce-docklike-plugin
cd /home/$USER/programs_setup/xfce-docklike-plugin
git clone https://github.com/nsz32/docklike-plugin && cd docklike-plugin
./autogen.sh
make
sudo make install
xfce4-panel -r&
echo "now rightclick the panel > panel preferences > items >"
echo "now add in panel 'Docklike Taskbar' below 'Window Buttons' and remove 'Window Buttons'"`
